### ISSUE TITLE
Integration setting for low-gain traces in CTA (e.g. LST)

### ISSUE THREAD

**Issue by GernotMaier** [Eventdisplay/Eventdisplay#57]
Integration window set occasionally wrong for low gain.
see e.g., 

```
./bin/evndisp -sourcefile ./Data/prod5/DST/S.BL-4LSTs00MSTs00SSTs-MSTF-gamma_cone-Emin05TeV.root -imagesquared -averagetzerofiducialradius=0.5 -arraycuts EVNDISP.prod5.reconstruction.runparameter -display=1 -highres
```

![Screenshot 2020-10-21 at 08 27 38](https://user-images.githubusercontent.com/14001688/96681278-484d6600-1377-11eb-8f6f-c0b0804de5a5.png)

**Comment by GernotMaier** [comment]
Related to peak finder in low-gain:

- peak finder optimised for VERITAS low-gain, where for a small period of time the end of the high-gain pulse was visible in the low-gain window
- reverse search introduced for this, which fails for CTA with pulses with a small afterpulse

Solution add option to switch off reverse integration.

```
 trace integration start
 (for window 1 for integration method 1)
    Par1: <sample # of window start>
    Par2: <shift of window start for double pass>
    Par3: <timing method used for calculation of window start (FIXED/TZERO/TAVERAGE)>
    Par4: <max difference between T0 and trace integration start>
    Par5: <max trace sample for sliding window search (default: nsamples)>
    Par6: <reverse search for low gain (default: 1)>
* -387      FADCSUMMATIONSTART 1 0 TAVERAGE -4 -1 0
* 10408618  FADCSUMMATIONSTART 10 0 TAVERAGE -4 95 0
* 10608418  FADCSUMMATIONSTART 0  0 TAVERAGE -4 -1 0
* 201409917 FADCSUMMATIONSTART 0  0 TAVERAGE -4 -1 0
```

New parameter fSumWindow_searchmaxreverse in inc/VEvndispRunParameter.h

**Comment by GernotMaier** [comment]
Fixed with commit 2fe1a6a , branch prod5-v08
